src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -62,6 +63,7 @@ public void generatePage() {
 
     private void generateReport() {
         context.put("report_file", getWebPage());
+        context.put("escHtml", StringEscapeUtils.ESCAPE_HTML4);
```

Comment:
No, I just setting a variable that allows me to use a function to escape characters into html entities in any velocity temple.